src/widget/wbeatspinbox.h

```diff
@@ -31,6 +31,15 @@ class WBeatSpinBox : public QDoubleSpinBox, public WBaseWidget {
 
     ControlProxy m_valueControl;
 
+    // for L/R button BeatJump/LoopMove
+    ControlProxy m_beatJumpForward;
+    ControlProxy m_beatJumpBackward;
+    ControlProxy m_beatJumpOneForward;
+    ControlProxy m_beatJumpOneBackward;
+    ControlProxy m_beatJumpEightForward;
+    ControlProxy m_beatJumpEightBackward;
+    ControlProxy m_beatLoopEnabled;
```

Comment:
We usually use pointers, specifically `parented_ptr` (src/util/parented_ptr.h). You'll find many examples in the codebase for how to use ad create those.